graphql/build.gradle

```diff
@@ -3,6 +3,9 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor mnSerde.micronaut.serde.processor
```

Comment:
should not this be?

```suggestion
    testAnnotationProcessor(mnSerde.micronaut.serde.processor)
```

Reply:
No, Serde also needs to be in the main source classpaths. `GraphQLApolloWsResponse` is `@Serdeable` because it is serialized at https://github.com/micronaut-projects/micronaut-graphql/pull/354/files#:~:text=kaMessage%20%3D%20graphQLJsonSerializer.serialize(new%20GraphQLApolloWsResponse(GQL_CONNECTION_KEEP_ALIVE))%3B

Otherwise, the example app `graphql-example-jwt-security` fails with 

```
Message: Error serializing object to JSON: No serializable introspection present for type GraphQLApolloWsResponse. Consider adding Serdeable. Serializable annotate to type GraphQLApolloWsResponse. Alternatively if you are not in control of the project's source code, you can use @SerdeImport(GraphQLApolloWsResponse.class) to enable serialization of this type.
Path Taken: new GraphQLApolloWsKeepAlive(WebSocketBroadcaster broadcaster,GraphQLApolloWsState state,GraphQLJsonSerializer graphQLJsonSerializer)
io.micronaut.context.exceptions.BeanInstantiationException: Error instantiating bean of type  [io.micronaut.configuration.graphql.apollo.ws.GraphQLApolloWsKeepAlive]
```